### DISCUSSION TITLE
Using google/matcha-base from Huggingface for combining several charts into one?

### DISCUSSION THREAD

**Discussion by PaulJKathmann** [community/community#57822] [Programming Help]
I am currently thinking of building a prototype for my university course, that allows me to combine several charts into a single chart and to compare charts (e.g. I have 3 charts each with the GDP per Capita of a country over time and want to overlay the charts into one and ask which one grew the fastest). 

I looked at this model https://huggingface.co/google/matcha-base and similar ones (https://huggingface.co/google/matcha-chartqa#get-predictions-from-the-model) was wondering if it's also possible to combine several charts into a single input?

And has anyone build a similar product, yet? I couldn't find any public projects using this model.

**Comment by sethjuarez** [comment]
I'm not sure I understand completely - is the goal to take images of charts to extract information and then recombine them?
